--- build_tools/menuselect-deps.in	(Asterisk 13.38.3)
+++ build_tools/menuselect-deps.in	(working copy)
@@ -6,2 +6,3 @@
 BISON=@PBX_BISON@
+BROADVOICE=@PBX_BROADVOICE@
 CURL=@PBX_CURL@
--- configure.ac	(Asterisk 13.38.3)
+++ configure.ac	(working copy)
@@ -478,2 +478,3 @@
 AST_EXT_LIB_SETUP([BLUETOOTH], [Bluetooth], [bluetooth])
+AST_EXT_LIB_SETUP([BROADVOICE], [BroadVoice Audio Decoder/Encoder], [broadvoice])
 AST_EXT_LIB_SETUP([CAP], [POSIX 1.e capabilities], [cap])
@@ -2561,2 +2562,4 @@
 
+AST_EXT_LIB_CHECK([BROADVOICE], [broadvoice], [bv32_encode], [broadvoice.h])
+
 AST_EXT_LIB_CHECK([SPEEX], [speex], [speex_encode], [speex/speex.h], [-lm])
--- makeopts.in	(Asterisk 13.38.3)
+++ makeopts.in	(working copy)
@@ -137,2 +137,5 @@
 
+BROADVOICE_INCLUDE=@BROADVOICE_INCLUDE@
+BROADVOICE_LIB=@BROADVOICE_LIB@
+
 CURL_INCLUDE=@CURL_INCLUDE@
